contracts/0.8.25/vaults/predeposit_guarantee/PredepositGuarantee.sol

```diff
@@ -118,6 +118,7 @@ contract PredepositGuarantee is CLProofVerifier, PausableUntilWithRoles {
         uint64 _changeSlot
     ) CLProofVerifier(_gIFirstValidator, _gIFirstValidatorAfterChange, _changeSlot) {
         _disableInitializers();
+        _pauseUntil(PAUSE_INFINITELY);
```

Comment:
So, now we need to explicitly `resume` the contract before usage, correct? Should not we resume it in initialize?